internal/hcsoci/diag.go

```diff
@@ -32,5 +32,8 @@ func ExecInUvm(ctx context.Context, vm *uvm.UtilityVM, req *shimdiag.ExecProcess
 	cmd.Stderr = np.Stderr()
 	cmd.Log = log.G(ctx).WithField(logfields.UVMID, vm.ID())
 	err = cmd.Run()
+	if err != nil {
+		return -1, err
```

Comment:
Why?

Reply:
Updated to a more appropriate fix in cmd.go

Reply:
Ok cool, this was confusing me

Reply:
Resolving